code/modules/antagonists/blob/blob_abilities.dm

```diff
@@ -435,7 +437,7 @@
 			return
 		if (!tutorial_check("consume", T))
 			return
-		playsound(T, 'sound/impact_sounds/Slimy_Hit_4.ogg', 50, 1)
+		owner.playsound_local(owner.loc, "sound/voice/blob/blob[rand(2, 4)].ogg", 60, 1)
```

Comment:
why 2 and 4

Reply:
because those are the sounds that fit it best out of the blobbuild sounds